src/platform/armv7a/boot_comp.c

```diff
@@ -19,6 +19,8 @@ struct cap_asnd hw_asnd_caps[HW_IRQ_TOTAL];
 void *thd_mem[NUM_CPU], *tcap_mem[NUM_CPU];
 struct captbl *glb_boot_ct;
 
+thdid_t tid = 0;
```

Comment:
Comment with a `TODO` to make additions to this atomic. This feels like it should *really* be `static`.